sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -112,3 +112,38 @@
     'WriteToJson': 'beam:schematransform:org.apache.beam:json_write:v1'
   config:
     gradle_target: 'sdks:java:extensions:schemaio-expansion-service:shadowJar'
+
+- type: renaming
+  transforms:
+    'ReadFromJdbc': 'ReadFromJdbc'
+    'WriteToJdbc': 'WriteToJdbc'
+  config:
+    mappings:
+      'ReadFromJdbc':
+        driver_class_name: 'driverClassName'
+        jdbc_url: 'jdbcUrl'
+        username: 'username'
+        password: 'password'
+        table_name: 'location'
+        read_query: 'readQuery'
+        num_rows: 'fetchSize'
+        driver_jars: 'driverJars'
+        connection_properties: 'connectionProperties'
+        connection_init_sql: 'connectionInitSql'
+      'WriteToJdbc':
+        driver_class_name: 'driverClassName'
+        jdbc_url: 'jdbcUrl'
+        username: 'username'
+        password: 'password'
+        table_name: 'location'
+        write_statement: 'writeStatement'
```

Comment:
What does this mean? Is it needed for the common case?

Reply:
I also debated this one...

The transform will loop over the `Row` fields and append them the a pre-written `INSERT` statement, i.e.
`INSERT INTO {table} VALUES ({field1}?, {field2}?, {fieldN}?)` which means that the function that feeds into it HAS to ensure the order is correct. Allowing `write_statement` would allow the user to supply a custom `INSERT` statement with named fields so they can supply the row with the columns in any order.

Reply:
Can we instead construct the INSERT statement from the Row schema itself so it always has the "right" order?

Reply:
The problem is the SchemaTransform does not know the table schema, so we can't set the order of the values in the INSERT statement based off the table schema. We could theoretically do a `INSERT INTO {table} (?, ?, ?) VALUES (?, ?, ?)`, but the current `BeamRowPreparedStatementSetter` does not support specifying columns (It just loops of the field values and replaces the ?'s iteratively), so it would require a major rewrite that would unlikely be completed by 2.52

I think that this might be worth another FR, but not blocking this PR.

Reply:
So I'd rather not go out with the `write_statement` option 'cause it'd be hard to remove later. The right thing to do is clear, but if that's not possible for 2.52 we either declare this transform usable without an explicit write_statement or we defer adding it 'till 2.53. (Looking at this, `BeamRowPreparedStatementSetter` seems to always be instantiated at a place where we know the schema, so it shouldn't be too hard to fix.)

Reply:
Actually, looking into this a bit more, we're constructing the default query right here: https://github.com/apache/beam/blob/e78a01a58f7b1e5894872217b2474fc84dea9956/sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcWriteSchemaTransformProvider.java#L104

Can't we use the names of the schema to construct a query

`INSERT INTO {table} (field1, field2, field3, ...) VALUES ...`

Yes, this would mean that names in your schema would have to agree with the names of your table, but this seems to be very much the right (and safe) thing to do.

Reply:
Would this possibly introduce a security risk exposing the column names directly in the query? I know we use the prepared statement for the values to protect against SQL injection since the values could possibly come from an unverified source. I think this is prevented for the column names since they are set in the row schema when the pipeline is written, so it should be safe, but do you think there could be a way to break this?

Reply:
I went ahead and updated it just in case

Reply:
Thanks. Yes, only the pipeline author can control the column names themselves.